modules/GKB/registry.xml

```diff
@@ -1,6 +1,6 @@
 <MartRegistry>
-  <MartURLLocation database="ensembl_mart_94" default="1" displayName="ENSEMBL GENES (SANGER UK)" host="useast.ensembl.org" includeDatasets="" martUser="" name="ENSEMBL_MART_ENSEMBL" path="/biomart/martservice" port="80" serverVirtualSchema="default" visible="1" />
-  <MartURLLocation database="plants_mart_41" default="" displayName="Plant Mart" host="plants.ensembl.org" includeDatasets="" martUser="" name="plants_mart" path="/biomart/martservice" port="80" serverVirtualSchema="plants_mart" visible="1" />  
-  <MartURLLocation database="protists_mart_41" default="" displayName="Protist Mart" host="protists.ensembl.org" includeDatasets="" martUser="" name="protist_mart" path="/biomart/martservice" port="80" serverVirtualSchema="protists_mart" visible="1" />
-  <MartURLLocation database="fungi_mart_41" default="" displayName="Fungal Mart" host="fungi.ensembl.org" includeDatasets="" martUser="" name="fungal_mart" path="/biomart/martservice" port="80" serverVirtualSchema="fungi_mart" visible="1" />
-</MartRegistry>    
+  <MartURLLocation database="ensembl_mart_95" default="1" displayName="ENSEMBL GENES (SANGER UK)" host="useast.ensembl.org" includeDatasets="" martUser="" name="ENSEMBL_MART_ENSEMBL" path="/biomart/martservice" port="80" serverVirtualSchema="default" visible="1" />
```

Comment:
Did these values have to be checked and updated manually?

Reply:
No, there's a function in the EnsEMBLMartUtls.pm module that updates these, but the file itself is there so it exists with the repo.  I can look into making something that creates it rather than keeping it in GitHub, but I'm not sure if we're going to keep the Other Identifiers step long term.

Reply:
Thats ok, I was curious because that meant you were the holder of some knowledge important to the release process. If it's automated I think its good